ffi/util.lua

```diff
@@ -669,10 +677,10 @@ local function orderedNext(t, state)
     -- We use a temporary ordered key table that is stored in the table being iterated.
 
     local key = nil
-    --print("orderedNext: state = "..tostring(state) )
+    --print("orderedNext: state = "..tostring(state))
```

Comment:
`--print` or `-- print`?

Reply:
That's a good question, do we have a consensus for that in the code?

(I rarely do that in Lua, but I tend to prefer "sticky" comments for those in other languages, so I'd keep it as-is myself, FWIW).

Reply:
I prefer with a space because I consider it more legible, so I suppose it's a stalemate.

Anyway, it's one of those things I'd have happily ignored for decades but you had to touch the line… :-P

Reply:
(I also prefer with a space after `--`, which is how 99% of our comments are.)

Reply:
That's perfectly fine by me ;o).